templates/Shop/Product/Show/_price.html.twig

```diff
@@ -14,10 +14,10 @@
     {% set days = sylius.channel.lowestPriceForDiscountedProductsCheckingPeriod %}
 
 <div class="ui mini label" id="product-lowest-price-before-the-discount"
-    {% if not has_discount and not variant|sylius_has_lowest_price({'channel': sylius.channel}) %}
-        style="display: none;"
-    {% else %}
+    {% if has_discount and variant|sylius_has_lowest_price({'channel': sylius.channel}) %}
```

Comment:
`variant|sylius_has_lowest_price` should be extracted to a var similarly how it's done w/ `has_discount`